service/cloudconfig/template.go

```diff
@@ -16,12 +15,13 @@ vnetName: {{ .VnetName }}
 routeTableName: {{ .RouteTableName }}
 useManagedIdentityExtension: true
 `
+)
 
-	// Key Vault secrets constants
-	keyVaultSecretsFileOwner      = "root:root"
-	keyVaultSecretsFilePermission = 0700
-	getKeyVaultSecretsFileName    = "/opt/bin/get-keyvault-secrets"
-	getKeyVaultSecretsTemplate    = `#!/bin/bash -e
+const (
+	getKeyVaultSecretsFileName       = "/opt/bin/get-keyvault-secrets"
+	getKeyVaultSecretsFileOwner      = "root:root"
+	getKeyVaultSecretsFilePermission = 0700
+	getKeyVaultSecretsFileTemplate   = `#!/bin/bash -e
```

Comment:
I split each file with new `const (`. Then you can search for `const (` and new line after that is a file name.

Some names are aligned. `keyVaultSecretsFileOwner` is renamed to `getKeyVaultSecretsFileOwner`. I also renamed things like `getKeyVaultSecretsTemplate` to `getKeyVaultSecretsFileTemplate`.